iroh-gateway/src/rpc.rs

```diff
@@ -1,20 +1,36 @@
-use std::result;
+use std::{result, time::Duration};
 
 use anyhow::Result;
+use futures::stream::Stream;
 use iroh_rpc_client::{create_server, GatewayServer, ServerError, ServerSocket};
-use iroh_rpc_types::gateway::{
-    GatewayAddr, GatewayRequest, GatewayService, VersionRequest, VersionResponse,
+use iroh_rpc_types::{
+    gateway::{GatewayAddr, GatewayRequest, GatewayService},
+    VersionRequest, VersionResponse, WatchRequest, WatchResponse,
 };
 use tracing::info;
 
+const VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
just realized, this will be different than the version we use in metrics, can you make an issue to update this to match please?

Reply:
I moved the definition to `lib.rs`, and referenced it in both the `metrics.rs` and `rpc.rs` files for each of `iroh-p2p`, `iroh-gateway`, and `iroh-store`